cardano-client-demo/MainPipeline.hs

```diff
@@ -0,0 +1,146 @@
+{-# LANGUAGE UndecidableInstances #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -freduction-depth=0 #-}
+
+import Cardano.Api
+    ( Block(..),
```

Comment:
We have a `stylish-haskell` config file in the top level that you should use: https://github.com/input-output-hk/cardano-node/blob/master/.stylish-haskell.yaml